setup.py

```diff
@@ -103,7 +103,8 @@ def run(self):
             'flocker-deploy = flocker.cli.script:flocker_deploy_main',
             'flocker-changestate = flocker.node.script:flocker_changestate_main',
             'flocker-reportstate = flocker.node.script:flocker_reportstate_main',
-            'flocker-serve = flocker.node.script:flocker_serve_main',
```

Comment:
There's still a functional test for `flocker-serve` existing which is now failing.  It should be replaced with tests for the two differently named scripts being introduced here, I guess.